static/app/components/smartSearchBar/index.tsx

```diff
@@ -583,7 +682,11 @@ class SmartSearchBar extends Component<Props, State> {
         searchGroups[groupIndex].children[childrenIndex];
 
       if (item) {
-        this.onAutoComplete(item.value, item);
+        if (item.callback) {
+          item.callback();
+        } else if (typeof item.value !== 'undefined') {
```

Comment:
do we need this `else if` here, could it just be `else`? or is there a case where an item has no callback and no value?

Reply:
just mainly for typescript because the quick actions don't have values and it could be undefined